File: ws/sanchez/u2/ws16Libraries/TaxesLib/src/utils/Tax.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package utils;

/**
 *
 * @author Joel Sanchez <The_Softwarriors at ESPE>
 */
public class Tax {
    public static float computeIva(float amount, float taxPorcentage){
       float taxValue;
       taxValue = amount * taxPorcentage/100;
       return taxValue;
    }
    
    public static float computeTotal(float amount, float taxPorcentage){
        float totalValue;
        totalValue = amount + computeIva(amount, taxPorcentage);
        return totalValue;
    }
}

File: hw/sanchez/u2/hw09Libraries/TaxesLibPY/utils/Tax.py
#author Joel Sanchez <The_Softwarriors at ESPE>

class Tax:
    @staticmethod
    def computeIva(amount, taxPorcentage):
        taxValue = amount * taxPorcentage / 100
        return taxValue
    
    @staticmethod
    def computeTotal(amount, taxPorcentage):
        totalValue = amount + Tax.computeIva(amount, taxPorcentage)
        return totalValue